mava/components/jax/building/parameter_client.py

```diff
@@ -116,13 +116,13 @@ def on_building_executor_parameter_client(self, builder: SystemBuilder) -> None:
 
         builder.store.executor_counts = {name: params[name] for name in count_names}
 
-        set_keys = get_keys.copy()
-
         # Executors should only be able to update relevant params.
         if builder.store.is_evaluator is True:
```

Comment:
Maybe replace this with:
```suggestion
        executor_type = "evaluator" if builder.store.is_evaluator else "executor"
        set_keys = [x for x in get_keys if x.startswith(executor_type)]
        get_keys = [x for x in get_keys if not x.startswith(executor_type)]
```

Reply:
Good idea